### DISCUSSION TITLE
3 Level scale

### DISCUSSION THREAD

**Discussion by cgolankotter** [twilio-labs/paste#3238] [General] [answered]
### Background

Hi Team,
We’re looking to create a 3 level protection scale/slider for customers to rate their risk tolerance.
Do you have a recommendation / guidance on what existing component could fit?
Thanks!

### Feedback

Recommendation / guidance on what component could fit. Thank!

### Relevant links

https://www.figma.com/proto/bKVSufe96rMBoFkXGEbudn/2023-Q2---SMS-Flooding?page-id=1%3A3900&node-id=6-116274&viewport=-1321%2C127%2C0.44&scaling=min-zoom&starting-point-node-id=6%3A116274

### Are you looking for visual design support?

- [ ] Yes

**Comment by loreina** [comment, marked as answer]
Hi @cgolankotter,

This use case matches a Slider component that we have on our roadmap for our August 8, 2023 release candidate. Could you file a [dependency request](https://airtable.com/shrvMvL3BG6Zm0f9l) for the Slider? 

In the meantime, you could look at a a treatment with [Radio Group](https://paste.twilio.design/components/radio-group) or [Radio Button Group](https://paste.twilio.design/components/radio-button-group), or have your engineering team build the component and swap to Slider when the component is available in Paste. Thanks!

**Reply by cgolankotter** [reply]
Thanks so much @loreina🙏! 
We'll use the Radio Button Group until the Slider is available.
